amida-tech/indaba-client
# Feature/inba 952 jf box removal

#### What does this PR do?
Removes the box component. Note that `Menu` items have started throwing warnings. However, this component will be removed with another ticket, INBA-955.

#### Related JIRA tickets:
INBA-952
INBA-955

#### How should this be manually tested?
Go to test the following features and ensure they look visually correct and continue to function:

1. Go through the Create Project Wizard and rigorously test the "Create Stage" section.
2. In a Project, check and verify that the "Project" and "Subject" modal popups still appear and work.
3. Check that the "Stage Summary" boxes in the "Project" workflow look and work as normal.
4. Try dragging and dropping users onto the workflow, and inviting them using the bottom right "User Invite" features.
5. Test the "Discussions". 

#### Background/Context

#### Screenshots (if appropriate):
